client/app/scripts/liveblog-edit/module.js

```diff
@@ -430,6 +435,44 @@ define([
         'superdesk.upload',
         'liveblog.pages-manager'
     ]);
+    app.service('blogSecurityService',
+        ['$q', '$rootScope', '$route', 'blogService', '$location', function($q, $rootScope, $route, blogService, $location) {
+        function isUserOwner() {
+            var def = $q.defer();
+            blogService.get($route.current.params._id)
+            .then(function(response) {
+                if ($rootScope.currentUser._id !== response.original_creator) {
+                    def.resolve(false);
+                } else {
+                    def.resolve(true);
+                }
+            }, function(response) {
+                if (response.status === 404) {
+                    def.reject('Blog does not exist');
+                }
+            });
+            return def.promise;
+        }
+        function canGoToSettings() {
```

Comment:
Methods like __can__Something, or __is__Something, shouldn't do any other action than returning a boolean. Here you redirect users to other route.